packages/flutter/lib/src/material/list_tile.dart

```diff
@@ -441,11 +457,16 @@ class ListTile extends StatelessWidget {
     return new InkWell(
       onTap: enabled ? onTap : null,
       onLongPress: enabled ? onLongPress : null,
-      child: new Container(
-        height: tileHeight,
-        padding: const EdgeInsets.symmetric(horizontal: 16.0),
-        child: new Row(children: children),
-      )
+      child: new ConstrainedBox(
+        constraints: new BoxConstraints(minHeight: tileHeight),
+        child: new Padding(
+          padding: const EdgeInsets.symmetric(horizontal: 16.0),
+          child: new Column(
```

Comment:
I don't understand why this Column is needed.

Reply:
Because without it, the row expands vertically to fill the entire available space now that it is unconstrained in height.

This one is what happens with the Column (for a ListTile inside of a Center):
![screenshot_20171013-145340](https://user-images.githubusercontent.com/8867023/31568020-1f6050f6-b027-11e7-8a06-aff16f774baf.png)

And this is what happens without the Column (also a ListTile inside of a Center):
![screenshot_20171013-145403](https://user-images.githubusercontent.com/8867023/31568019-1f477ba8-b027-11e7-9093-5e3b0438c314.png)

If you never expect ListTiles to be used outside of a shrink-wrapped container, then the Column might not be needed.

Reply:
We should find a better solution. Having a Column for a single child is not the optimal solution.

Reply:
(Presumably that's UnconstrainedBox?)